src/components/layouts/collection-page-layout.tsx

```diff
@@ -1094,4 +1204,92 @@ const transformSanityTopics = (topics: any) => {
   return topics.items
 }
 
+const DialogButton = ({title, buttonStyles, buttonText, children}: any) => {
```

Comment:
suggestion [code style]: this is a monster above and I think that the DialogButton is likely generally useful so I'd recommend extracting it out into its own component file

Reply:
Should I move it to components/buttons/index.tsx?